src/firefly/js/ui/panel/TabPanel.css

```diff
@@ -45,3 +47,9 @@
     display: flex;
 }
 
+.ellipsis {
+    display: inline-block;
```

Comment:
I don't think so. It's not a modifier. It is quite generic style for text: if there is not enough room, the text is clipped and the clipped part is represented with '...' 

I will rename it to text-ellipsis and move it to global.css